busylight/api/busylight_api.py

```diff
@@ -113,6 +143,17 @@ async def apply_effect(self, effect: Effects, light_id: int = None) -> None:
 
     def get(self, path: str, **kwargs) -> Callable:
         self.endpoints.append(path)
+
+        # CORS allowed origins (for the Access-Control-Allow-Origin header)
+        # are set through an environment variable BUSYLIGHT_API_CORS_ORIGINS_LIST
+        # e.g.: export BUSYLIGHT_API_CORS_ORIGINS_LIST='["http://localhost", "http://localhost:8080"]'
+        # (see https://fastapi.tiangolo.com/tutorial/cors/ for details)
+        if self.origins != None:
```

Comment:
I prefer to leave tests like this a little more ambiguous and depend on the "truthiness" of objects:
```
if self.origins:
```

If `self.origins` is an empty list, it will evaluate False, otherwise True.